app/build.gradle

```diff
@@ -1,5 +1,19 @@
 apply plugin: 'com.android.application'
 
+repositories {
+    maven {
+        // Choose whatever name you like
+        name = "GitHubPackages"
+        // The url of the repository that contains the published artifacts
+        url = "https://maven.pkg.github.com/minima-global/Minima"
+        credentials {
+            // The credentials (described in the next section)
+            username = project.findProperty("gpr.user")
```

Comment:
I changed this to env var in the PR#2. This may cause a merge conflict. 
I'd suggest the easiest solution, if you happy with both PR's to just merge #2 and close this one